src/unstract/adapters/embedding/ollama/README.md

```diff
@@ -0,0 +1 @@
+# Unstract Ollama Embedding Adapter
```

Comment:
Just a thought, Atleast we can add some ref docs here. For this, we can add ollama docs you reffered. (Not mandatory now)

Reply:
@muhammad-ali-e I have addressed this comment to some extent. Pls check and approve if that is ok.